.github/workflows/dist.yml

```diff
@@ -35,6 +35,11 @@ env:
   COLUMNS: "100"
   FORCE_COLOR: "1"
   CLICOLOR_FORCE: "1"
+  UV_INDEX_STRATEGY: "unsafe-best-match"
```

Comment:
See comment about `UV_INDEX` below.

Reply:
`UV_INDEX` combined with `UV_INDEX_STRATEGY: "unsafe-best-match"` is more consistent with the default behavior with `pip`.

If you are using `UV_INDEX` with default `UV_INDEX_STRATEGY: "first-index"`, that can cause problems. See also:

- pytorch/pytorch#156694